index.js

```diff
@@ -7,14 +7,180 @@ function ask(questionText) {
+  false,
+  false
+)
+
+let kitchen = new Room(
+  "kitchen",
+  "Craft delicious delicacies to satiate yourself here!",
+  "You can enter the HALLWAY or the LOUNGE or the FRIDGE from here.",
+  "This room contains a FRIDGE!",
+  false
+)
+
+let lounge = new Room(
+  "lounge",
+  "Sit back, kick your feet up, and relax. You've earned it!",
+  "You can enter the KITCHEN from here!",
+  "This room contains a KNIFE!",
+  false
+)
+
+let fridge = new Room(
+  "fridge",
+  "It's cold but all your food is preserved in here!",
+  "You can enter the KITCHEN from here!",
+  "This room contains a ",
+  false
+  )
+
+let local = bedroom // Defines starting location
+
+let lookupLocal = {
+  bedroom: bedroom,
+  hallway: hallway,
+  kitchen: kitchen,
+  lounge: lounge,
+  fridge: fridge
+}
+
+let localTrans = { //Creates object that determines which rooms have access to others
+  bedroom: ["hallway"],
+  hallway: ["bedroom", "kitchen"],
+  kitchen: ["hallway", "lounge", "fridge"],
+  lounge: ["kitchen"],
+  fridge: ["kitchen"]
 }
+
+function move(target) { //function that is supposed to allow you to move rooms
+  if (localTrans[local.movement].includes(target)) {
+    local = lookupLocal[target]
+    console.log(local)
+  } else {
+    console.log(this.movement)
+  }
+}
+
+async function start() { //function to start the game and allow player inputs
+  let playerAction = await ask("What would you like to do? \n>_") //asks for player input
+  let inputArray = playerAction.toLowerCase().split(" ") //converts player input to lower case and splits by spaces
+  let action = inputArray[0] // defines action to the first word of the player input this will be used to access the action constructor of the item class
+  let target = inputArray.slice(1).join(" ") // defines target to the rest of the player input which will be used to reference previously defined rooms and items
+
+  if (action === "use") { //if first word of player input is use then game runs the use method for item
+    console.log(lookupItem[target].use());
+  } else if (action === "take") { //if first word of player input is take then game runs the take method for item, if player input is not in the item class then the game responds negatively
+    if (lookupItem[target] instanceof Item) {
+      console.log(lookupItem[target].take())
+    } else {
+      console.log(lookupItem[target] instanceof Item)
+      console.log("That's not an item!")
+    }
+  } else if (action === "examine") { // if first word of player input is examine then game will run the examineI method for item
+    console.log(lookupItem[target].examineI())
+  }
+  if (action === "move") { // if first word of player input is move then game runs the move function with the rest of the player input as the parameter.
+    move(target)
+    console.log(Room.desciption)
+    console.log(Room.movement)
+    console.log(Room.items)
+  }
+  return start() // loops back to that top 
+}
```

Comment:
this works pretty great, consider having an "else" catch that returns "i cant do that" or something similar so it doesn't just repeat "what would you like to do"